Add 1.19.4 Support

x64architecture opened the pull request:
Hi,

I added support for 1.19.4. I tested it under Fabric and Forge and so far and everything seems okay to me but I haven't used prior versions so I don't really have anything to base it off of.

I am new to working with Minecraft mods so some of the dependency updates may have been unnecessary or may make using lower versions (i.e. 1.19.3) impossible. Let me know if anything needs to be changed.

Potential Problems:

There was a bunch of Frog animation state stuff that was removed in 1.19.3+, I commented out the code but I am not sure what it was supposed to do. When using the frog the limbs properly move and when in water the frog properly does the swim animation, so perhaps they unified it to be like other entities and use generic classes.

The `airStrafingSpeed` property was removed from the RavagerEntity class, I could not figure out what the replacement is, or if there is one.

The tooltip stuff could have issues, though they seem to work, again not sure how they worked in prior versions.

If anyone wants to try this out for themselves I have binaries posted here:

(2023-05-24) Updated to fix Forge version:

https://github.com/x64architecture/identity/releases/tag/1.19.4-Dev-Build-2

x64architecture committed:
Add 1.19.4 Support

  * Architectury updated to 8.1.80
  * Architectury Loom updated to 1.2-SNAPSHOT
  * Gradle updated to 8.1.1
  * Shadow updated to 8.1.1
  * Omega-Config updated to 1.3.0+1.19.2
  * Targeting Fabric loader 0.14.19
  * Targeting Fabric API version 0.81.1+1.19.4
  * Targeting Forge version 45.0.64
  * Requiring Java 17+

0xicl33n commented:
Immediate crash when opening the identity menu on `forge-49.0.64`. 
```
---- Minecraft Crash Report ----
// Surprise! Haha. Well, this is awkward.

Time: 2023-05-23 09:41:59
Description: Unexpected error

org.spongepowered.asm.mixin.transformer.throwables.MixinTransformerError: An unexpected critical error was encountered
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:250) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.service.modlauncher.MixinTransformationHandler.processClass(MixinTransformationHandler.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.launch.MixinLaunchPluginLegacy.processClass(MixinLaunchPluginLegacy.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at cpw.mods.modlauncher.serviceapi.ILaunchPluginService.processClassWithFlags(ILaunchPluginService.java:156) ~[modlauncher-10.0.8.jar:10.0.8+10.0.8+main.0ef7e830] {}
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.6.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.6.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) ~[securejarhandler-2.1.6.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) ~[securejarhandler-2.1.6.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) ~[securejarhandler-2.1.6.jar:?] {}
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?] {}
	at net.minecraft.world.entity.monster.piglin.PiglinBrute.m_8075_(PiglinBrute.java:104) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:classloading}
	at net.minecraft.world.entity.LivingEntity.<init>(LivingEntity.java:241) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,re:computing_frames,re:classloading,pl:mixin:APP:identity.mixins.json:LivingEntityFoodMixin,pl:mixin:APP:identity.mixins.json:LivingEntityMixin,pl:mixin:APP:identity.mixins.json:PlayerSwimmingMixin,pl:mixin:APP:identity.mixins.json:accessor.LivingEntityAccessor,pl:mixin:A}
	at net.minecraft.world.entity.Mob.<init>(Mob.java:124) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:identity.mixins.json:accessor.MobEntityAccessor,pl:mixin:A}
	at net.minecraft.world.entity.PathfinderMob.<init>(PathfinderMob.java:14) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,re:computing_frames,re:classloading}
	at net.minecraft.world.entity.monster.Monster.<init>(Monster.java:31) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,re:computing_frames,re:classloading}
	at net.minecraft.world.entity.monster.piglin.AbstractPiglin.<init>(AbstractPiglin.java:32) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:classloading,re:mixin}
	at net.minecraft.world.entity.monster.piglin.PiglinBrute.<init>(PiglinBrute.java:73) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:classloading}
	at net.minecraft.world.entity.EntityType.m_20615_(EntityType.java:525) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,xf:fml:forge:forge_method_redirector,re:classloading,xf:fml:forge:forge_method_redirector,pl:mixin:APP:architectury-common.mixins.json:inject.MixinEntityType,pl:mixin:A}
	at draylar.identity.api.variant.IdentityType.getAllTypes(IdentityType.java:91) ~[identity-2.6.1-1.19.4-forge-dev-build-1.jar%23155!/:?] {re:mixin,re:classloading}
	at draylar.identity.screen.IdentityScreen.populateRenderEntities(IdentityScreen.java:221) ~[identity-2.6.1-1.19.4-forge-dev-build-1.jar%23155!/:?] {re:classloading}
	at draylar.identity.screen.IdentityScreen.<init>(IdentityScreen.java:55) ~[identity-2.6.1-1.19.4-forge-dev-build-1.jar%23155!/:?] {re:classloading}
	at draylar.identity.impl.tick.MenuKeyPressHandler.tick(MenuKeyPressHandler.java:17) ~[identity-2.6.1-1.19.4-forge-dev-build-1.jar%23155!/:?] {re:classloading}
	at draylar.identity.impl.tick.MenuKeyPressHandler.tick(MenuKeyPressHandler.java:9) ~[identity-2.6.1-1.19.4-forge-dev-build-1.jar%23155!/:?] {re:classloading}
	at java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:732) ~[?:?] {re:mixin}
	at dev.architectury.event.EventFactory.invokeMethod(EventFactory.java:53) ~[architectury-8.1.80-forge.jar%23154!/:?] {re:classloading}
	at dev.architectury.event.EventFactory$1.handleInvocation(EventFactory.java:62) ~[architectury-8.1.80-forge.jar%23154!/:?] {re:classloading}
	at com.google.common.reflect.AbstractInvocationHandler.invoke(AbstractInvocationHandler.java:87) ~[guava-31.1-jre.jar%23105!/:?] {}
	at jdk.proxy3.$Proxy41.tick(Unknown Source) ~[?:?] {}
	at dev.architectury.event.forge.EventHandlerImplClient.event(EventHandlerImplClient.java:55) ~[architectury-8.1.80-forge.jar%23154!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at dev.architectury.event.forge.__EventHandlerImplClient_event_ClientTickEvent.invoke(.dynamic) ~[architectury-8.1.80-forge.jar%23154!/:?] {re:classloading,pl:eventbus:B}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.3.jar%2385!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.3.jar%2385!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.3.jar%2385!/:?] {}
	at net.minecraftforge.event.ForgeEventFactory.onPreClientTick(ForgeEventFactory.java:877) ~[forge-1.19.4-45.0.64-universal.jar%23161!/:?] {re:classloading}
	at net.minecraft.client.Minecraft.m_91398_(Minecraft.java:1745) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1114) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:719) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:205) ~[forge-45.0.64.jar:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:27) ~[fmlloader-1.19.4-45.0.64.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?] {}
Caused by: org.spongepowered.asm.mixin.throwables.MixinApplyError: Mixin [identity.mixins.json:PiglinBruteBrainMixin] from phase [DEFAULT] in config [identity.mixins.json] FAILED during APPLY
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinError(MixinProcessor.java:636) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.handleMixinApplyError(MixinProcessor.java:588) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:379) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	... 54 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InvalidInjectionException: Critical injection failure: Callback group @Group(name=method_30249FilterLambda, min=1, max=1) in identity.mixins.json:PiglinBruteBrainMixin failed injection check: maximum of 1 invocation(s) allowed but 2 succeeded [ -> PostApply Phase -> identity.mixins.json:PiglinBruteBrainMixin]
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.postApply(MixinTargetContext.java:1262) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:344) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:383) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:365) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	... 54 more
Caused by: org.spongepowered.asm.mixin.injection.throwables.InjectionValidationException: maximum of 1 invocation(s) allowed but 2 succeeded
	at org.spongepowered.asm.mixin.injection.struct.InjectorGroupInfo.validate(InjectorGroupInfo.java:270) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.injection.struct.InjectorGroupInfo$Map.validateAll(InjectorGroupInfo.java:126) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinTargetContext.postApply(MixinTargetContext.java:1255) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinApplicatorStandard.apply(MixinApplicatorStandard.java:344) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.apply(TargetClassContext.java:383) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.TargetClassContext.applyMixins(TargetClassContext.java:365) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:363) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	... 54 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at org.spongepowered.asm.mixin.transformer.MixinProcessor.applyMixins(MixinProcessor.java:392) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.mixin.transformer.MixinTransformer.transformClass(MixinTransformer.java:250) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.service.modlauncher.MixinTransformationHandler.processClass(MixinTransformationHandler.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at org.spongepowered.asm.launch.MixinLaunchPluginLegacy.processClass(MixinLaunchPluginLegacy.java:131) ~[mixin-0.8.5.jar:0.8.5+Jenkins-b310.git-155314e6e91465dad727e621a569906a410cd6f4] {}
	at cpw.mods.modlauncher.serviceapi.ILaunchPluginService.processClassWithFlags(ILaunchPluginService.java:156) ~[modlauncher-10.0.8.jar:10.0.8+10.0.8+main.0ef7e830] {}
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:88) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.TransformingClassLoader.maybeTransformClassBytes(TransformingClassLoader.java:50) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.readerToClass(ModuleClassLoader.java:113) ~[securejarhandler-2.1.6.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.lambda$findClass$15(ModuleClassLoader.java:219) ~[securejarhandler-2.1.6.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadFromModule(ModuleClassLoader.java:229) ~[securejarhandler-2.1.6.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.findClass(ModuleClassLoader.java:219) ~[securejarhandler-2.1.6.jar:?] {}
	at cpw.mods.cl.ModuleClassLoader.loadClass(ModuleClassLoader.java:135) ~[securejarhandler-2.1.6.jar:?] {}
	at java.lang.ClassLoader.loadClass(ClassLoader.java:520) ~[?:?] {}
	at net.minecraft.world.entity.monster.piglin.PiglinBrute.m_8075_(PiglinBrute.java:104) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:classloading}
	at net.minecraft.world.entity.LivingEntity.<init>(LivingEntity.java:241) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,re:computing_frames,re:classloading,pl:mixin:APP:identity.mixins.json:LivingEntityFoodMixin,pl:mixin:APP:identity.mixins.json:LivingEntityMixin,pl:mixin:APP:identity.mixins.json:PlayerSwimmingMixin,pl:mixin:APP:identity.mixins.json:accessor.LivingEntityAccessor,pl:mixin:A}
	at net.minecraft.world.entity.Mob.<init>(Mob.java:124) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,pl:accesstransformer:B,re:computing_frames,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:identity.mixins.json:accessor.MobEntityAccessor,pl:mixin:A}
	at net.minecraft.world.entity.PathfinderMob.<init>(PathfinderMob.java:14) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,re:computing_frames,re:classloading}
	at net.minecraft.world.entity.monster.Monster.<init>(Monster.java:31) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,re:computing_frames,re:classloading}
	at net.minecraft.world.entity.monster.piglin.AbstractPiglin.<init>(AbstractPiglin.java:32) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:classloading,re:mixin}
	at net.minecraft.world.entity.monster.piglin.PiglinBrute.<init>(PiglinBrute.java:73) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:classloading}
	at net.minecraft.world.entity.EntityType.m_20615_(EntityType.java:525) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,xf:fml:forge:forge_method_redirector,re:classloading,xf:fml:forge:forge_method_redirector,pl:mixin:APP:architectury-common.mixins.json:inject.MixinEntityType,pl:mixin:A}
	at draylar.identity.api.variant.IdentityType.getAllTypes(IdentityType.java:91) ~[identity-2.6.1-1.19.4-forge-dev-build-1.jar%23155!/:?] {re:mixin,re:classloading}
	at draylar.identity.screen.IdentityScreen.populateRenderEntities(IdentityScreen.java:221) ~[identity-2.6.1-1.19.4-forge-dev-build-1.jar%23155!/:?] {re:classloading}
	at draylar.identity.screen.IdentityScreen.<init>(IdentityScreen.java:55) ~[identity-2.6.1-1.19.4-forge-dev-build-1.jar%23155!/:?] {re:classloading}
	at draylar.identity.impl.tick.MenuKeyPressHandler.tick(MenuKeyPressHandler.java:17) ~[identity-2.6.1-1.19.4-forge-dev-build-1.jar%23155!/:?] {re:classloading}
	at draylar.identity.impl.tick.MenuKeyPressHandler.tick(MenuKeyPressHandler.java:9) ~[identity-2.6.1-1.19.4-forge-dev-build-1.jar%23155!/:?] {re:classloading}
	at java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:732) ~[?:?] {re:mixin}
	at dev.architectury.event.EventFactory.invokeMethod(EventFactory.java:53) ~[architectury-8.1.80-forge.jar%23154!/:?] {re:classloading}
	at dev.architectury.event.EventFactory$1.handleInvocation(EventFactory.java:62) ~[architectury-8.1.80-forge.jar%23154!/:?] {re:classloading}
	at com.google.common.reflect.AbstractInvocationHandler.invoke(AbstractInvocationHandler.java:87) ~[guava-31.1-jre.jar%23105!/:?] {}
	at jdk.proxy3.$Proxy41.tick(Unknown Source) ~[?:?] {}
	at dev.architectury.event.forge.EventHandlerImplClient.event(EventHandlerImplClient.java:55) ~[architectury-8.1.80-forge.jar%23154!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at dev.architectury.event.forge.__EventHandlerImplClient_event_ClientTickEvent.invoke(.dynamic) ~[architectury-8.1.80-forge.jar%23154!/:?] {re:classloading,pl:eventbus:B}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:73) ~[eventbus-6.0.3.jar%2385!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:315) ~[eventbus-6.0.3.jar%2385!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:296) ~[eventbus-6.0.3.jar%2385!/:?] {}
	at net.minecraftforge.event.ForgeEventFactory.onPreClientTick(ForgeEventFactory.java:877) ~[forge-1.19.4-45.0.64-universal.jar%23161!/:?] {re:classloading}
-- Affected level --
Details:
	All players: 1 total; [LocalPlayer['Ph0sgene'/71, l='ClientLevel', x=469.78, y=61.00, z=636.48]]
	Chunk stats: 961, 609
	Level dimension: minecraft:overworld
	Level spawn location: World: (16,63,176), Section: (at 0,15,0 in 1,3,11; chunk contains blocks 16,-64,176 to 31,319,191), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 1253 game time, 1253 day time
	Server brand: forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.ClientLevel.m_6026_(ClientLevel.java:451) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,pl:runtimedistcleaner:A,re:classloading,pl:mixin:APP:architectury.mixins.json:MixinClientLevel,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91354_(Minecraft.java:2322) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:741) ~[client-1.19.4-20230314.122934-srg.jar%23156!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:205) ~[forge-45.0.64.jar:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:27) ~[fmlloader-1.19.4-45.0.64.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) ~[modlauncher-10.0.8.jar:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141) ~[bootstraplauncher-1.1.2.jar:?] {}


-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: vanilla, mod_resources

-- System Details --
Details:
	Minecraft Version: 1.19.4
	Minecraft Version ID: 1.19.4
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 1000345816 bytes (954 MiB) / 2323644416 bytes (2216 MiB) up to 4294967296 bytes (4096 MiB)
	CPUs: 24
	Processor Vendor: GenuineIntel
	Processor Name: 12th Gen Intel(R) Core(TM) i9-12900K
	Identifier: Intel64 Family 6 Model 151 Stepping 2
	Microarchitecture: Alder Lake
	Frequency (GHz): 3.19
	Number of physical packages: 1
	Number of physical CPUs: 16
	Number of logical CPUs: 24
	Graphics card #0 name: NVIDIA GeForce RTX 3080
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x2206
	Graphics card #0 versionInfo: DriverVersion=31.0.15.3179
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.13
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 2.13
	Memory slot #1 type: DDR4
	Memory slot #2 capacity (MB): 8192.00
	Memory slot #2 clockSpeed (GHz): 2.13
	Memory slot #2 type: DDR4
	Memory slot #3 capacity (MB): 8192.00
	Memory slot #3 clockSpeed (GHz): 2.13
	Memory slot #3 type: DDR4
	Virtual memory max (MB): 93957.05
	Virtual memory used (MB): 34520.23
	Swap memory total (MB): 61440.00
	Swap memory used (MB): 2413.15
	JVM Flags: 4 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx4096m -Xms256m
	Launched Version: forge-45.0.64
	Backend library: LWJGL version 3.3.1 build 7
	Backend API: NVIDIA GeForce RTX 3080/PCIe/SSE2 GL version 3.2.0 NVIDIA 531.79, NVIDIA Corporation
	Window size: 1024x768
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'forge'; Server brand changed to 'forge'
	Type: Integrated Server (map_client.txt)
	Graphics mode: fancy
	Resource Packs: 
	Current Language: en_us
	CPU: 24x 12th Gen Intel(R) Core(TM) i9-12900K
	Server Running: true
	Player Count: 1 / 8; [ServerPlayer['Ph0sgene'/71, l='ServerLevel[New World]', x=469.78, y=61.00, z=636.48]]
	Data Packs: vanilla, mod:architectury (incompatible), mod:identity (incompatible), mod:forge
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Experimental
	ModLauncher: 10.0.8+10.0.8+main.0ef7e830
	ModLauncher launch target: forgeclient
	ModLauncher naming: srg
	ModLauncher services: 
		mixin-0.8.5.jar mixin PLUGINSERVICE 
		eventbus-6.0.3.jar eventbus PLUGINSERVICE 
		fmlloader-1.19.4-45.0.64.jar slf4jfixer PLUGINSERVICE 
		fmlloader-1.19.4-45.0.64.jar object_holder_definalize PLUGINSERVICE 
		fmlloader-1.19.4-45.0.64.jar runtime_enum_extender PLUGINSERVICE 
		fmlloader-1.19.4-45.0.64.jar capability_token_subclass PLUGINSERVICE 
		accesstransformers-8.0.4.jar accesstransformer PLUGINSERVICE 
		fmlloader-1.19.4-45.0.64.jar runtimedistcleaner PLUGINSERVICE 
		modlauncher-10.0.8.jar mixin TRANSFORMATIONSERVICE 
		modlauncher-10.0.8.jar fml TRANSFORMATIONSERVICE 
	FML Language Providers: 
		minecraft@1.0
		lowcodefml@null
		javafml@null
	Mod List: 
		client-1.19.4-20230314.122934-srg.jar             |Minecraft                     |minecraft                     |1.19.4              |DONE      |Manifest: a1:d4:5e:04:4f:d3:d6:e0:7b:37:97:cf:77:b0:de:ad:4a:47:ce:8c:96:49:5f:0a:cf:8c:ae:b2:6d:4b:8a:3f
		forge-1.19.4-45.0.64-universal.jar                |Forge                         |forge                         |45.0.64             |DONE      |Manifest: 84:ce:76:e8:45:35:e4:0e:63:86:df:47:59:80:0f:67:6c:c1:5f:6e:5f:4d:b3:54:47:1a:9f:7f:ed:5e:f2:90
		architectury-8.1.80-forge.jar                     |Architectury                  |architectury                  |8.1.80              |DONE      |Manifest: NOSIGNATURE
		identity-2.6.1-1.19.4-forge-dev-build-1.jar       |Identity                      |identity                      |2.6.1-1.19.4        |DONE      |Manifest: NOSIGNATURE
	Crash Report UUID: 53f455a8-b8c3-48b2-bbf1-9550bd99034d
	FML: 45.0
	Forge: net.minecraftforge:45.0.64
	`

x64architecture committed:
Remove No Longer Needed m_35106_ Mixin for PiglinBruteBrain

This was needed because Forge didn't have method_30255 while Identity did, but now both of them have it so it is no longer necessary.

x64architecture committed:
Bump version to 2.7.0

x64architecture commented:
@0xicl33n I pushed a change to my branch to fix the issue with Forge. You can give it a try again if you want.

https://github.com/x64architecture/identity/releases/tag/1.19.4-Dev-Build-2

Natedogwoooof commented:
> @0xicl33n I pushed a change to my branch to fix the issue with Forge. You can give it a try again if you want.
> 
> https://github.com/x64architecture/identity/releases/tag/1.19.4-Dev-Build-2

heck yeah ill try it and see if your unoffical update works or has bugs

Natedogwoooof commented:
thank you this is my favorite mod

Natedogwoooof commented:
we got 1.19.3 now

0xicl33n commented:
> @0xicl33n I pushed a change to my branch to fix the issue with Forge. You can give it a try again if you want.
> 
> https://github.com/x64architecture/identity/releases/tag/1.19.4-Dev-Build-2

took me a while to get back to this. yeah it works great!
